components/dynatrace_activegate/prod.tf

```diff
@@ -1,7 +1,7 @@
-module "dynatrace_activegate" {
-  source         = "../../modules/dynatrace-activegate"
-  instance_count = 3
-  network_zone   = "azure.cft"
+# module "dynatrace_activegate" {
+#   source         = "../../modules/dynatrace-activegate"
+#   instance_count = 3
+#   network_zone   = "azure.cft"
```

Comment:
Hi @MatthewFox97 why is the network zone commented out?

Reply:
@cakeben - lets catch up tomorrow with this

Reply:
@MatthewFox97 ^